pycqed/measurement/composite_detector_functions.py

```diff
@@ -1349,12 +1361,12 @@ def acquire_data_point(self, *args, **kw):
         # Qubit
         print('Scanning for qubit.' +
               'range: {fmin} - {fmax} GHz   span {span} MHz'.format(
```

Comment:
Let's avoid using this hard coded unit conversion. There exists a unit aware string formatting in the analysis tools. I can show you where but it is used in most of the v2 analyses.